### PR TITLE
Change Twitter to X

### PR DESCRIPTION
**Pull request by sticks-stuff**
Requires https://github.com/QuiltMC/quilt-bulma/pull/13

---
See preview on Cloudflare Pages: https://preview-150.quiltmc-org.pages.dev

### CODE REVIEW

File: src/data/Site.ts [hunk 1]
```diff
@@ -70,15 +70,15 @@ const config: Config = {
...
+		handle: new SocialMediaHandle("https://x.com/", "quilt_mc"),
 		card: "summary",
 	},
 
 	social: {
 		name: "QuiltMC",
 
 		links: [
-			new URL("https://twitter.com/quilt_mc"),
+			new URL("https://x.com/quilt_mc"),
```
Comment (Southpaw1496) [inline src/data/Site.ts:81]:
```suggestion
			new URL("https://twitter.com/quilt_mc"),
```
x.com still redirects to twitter.com, changing the URL is pointless.

File: src/data/TeamData.mjs
Comment (Southpaw1496) [inline src/data/TeamData.mjs:1]: Changing x.com to twitter.com is pointless since x.com still redirects to twitter.com.

**Review by Southpaw1496 — APPROVED** [review]
Mostly good except that there's no point changing all of the URLs to x.com since x.com still redirects to twitter.com

**Review by Southpaw1496 — CHANGES_REQUESTED** [review]
Mostly good except that there's no point changing all of the URLs to x.com since x.com still redirects to twitter.com

File: src/data/Site.ts [hunk 2]
```diff
@@ -70,7 +70,7 @@ const config: Config = {
 	twitch: new SocialMediaHandle("https://twitch.tv/", "quiltmc"),
 
 	twitter: {
-		handle: new SocialMediaHandle("https://twitter.com/", "quilt_mc"),
+		handle: new SocialMediaHandle("https://x.com/", "quilt_mc"),
```
Comment (Southpaw1496) [inline src/data/Site.ts:73]: Missed one

**Review by Southpaw1496 — CHANGES_REQUESTED** [review]

**Comment by sticks-stuff** [comment]
Fixed, sorry

**Review by Southpaw1496 — APPROVED** [review]
If we're waiting on the Quilt Bulma PR it may be a little while: that's someone else's job 🙃

**Comment by sticks-stuff** [comment]
Ah, shame :(

**Comment by LambdAurora** [comment]
Just saw this PR for some reason.

I'd rather not use a brand that sounds and looks shady af, which is still not that much in people's mind aside of being a joke, to satisfy the ego of some random delusional person who's way too rich for their own good.

**Pull request closed without merging**